### ISSUE TITLE
Extra warnings generated from builders

### ISSUE THREAD

**Issue by powerdude** [pmrogala/Buildenator#67]
Seeing this warning for all uses of `?` in the builder code that should probably be addressed:

```
WARNING - obj\Debug\net8.0\Buildenator\Buildenator.BuildersGenerator\ExerciseUpdatedEventBuilder_11.cs (72, 63): CS8669: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context. Auto-generated code requires an explicit '#nullable' directive in source.
```

**Comment by pmrogala** [comment]
Have you tried to explicitly enable the nullable for builder generation?
Here's an example:
https://github.com/pmrogala/Buildenator/blob/bba5a3e6b732c90db228349575659521798c5545/Tests/Buildenator.IntegrationTests.SourceNullable/AssemblyInfo.cs#L6
If it does not help, it'd be great if you provide me an example to check

**Comment by pmrogala** [comment]
Is your test project using nullable?
It's quite weird there's this error, it usually appears when the entity is in the project with nullable enabled and the builder is in a project without nullable enabled.
It's hard to detect such thing, that's why I added option to enable it manually

**Comment by powerdude** [comment]
yes, my project is using nullable.  It shouldn't be necessary for the user to add a configuration like this to their file especially since the generated code can just include `#nullable` in the generated source.

I'll include the attribute, but I'd really suggest that it shouldn't be required.

**Comment by pmrogala** [comment]
I remember that it was a bit annoying to implement, but I'll double check that

**Comment by pmrogala** [comment]
Fixed with 8.2.1.0
